test/specs/scenarios/face.js

```diff
@@ -332,20 +369,50 @@ export const faceScenarios = (lang) => {
       driver.executeScript(
         'window.navigator.mediaDevices.enumerateDevices = () => Promise.resolve([{ kind: "video" }])'
       )
+      await takePercySnapshot(
+        driver,
+        `Verify Submit passport photo page does not have onfido logo ${lang}`
+      )
       documentUpload.clickUploadButton()
       uploadFileAndClickConfirmButton(
         passportUploadImageGuide,
         confirm,
         'passport.jpg'
       )
       faceVideoIntro.checkLogoIsHidden()
+      await takePercySnapshot(
+        driver,
+        `Verify Let’s make sure nobody’s impersonating you screen does not have onfido logo ${lang}`
+      )
       faceVideoIntro.clickOnContinueButton()
       camera.checkLogoIsHidden()
-      camera.recordVideo()
-      camera.completeChallenges()
+      camera.enableCameraButton().click()
+      await takePercySnapshotWithoutOverlay(
+        driver,
+        `Verify Position your face in the oval overlay screen does not have onfido logo ${lang}`
+      )
+      camera.recordButton().click()
+      await takePercySnapshotWithoutOverlay(
+        driver,
+        `Movement challenge is given ${lang}`
+      )
+      camera.nextChallengeButton().click()
+      await takePercySnapshotWithoutOverlay(
+        driver,
+        `Vocal challenge is given ${lang}`
+      )
+      camera.stopButton().click()
       confirm.checkLogoIsHidden()
+      await takePercySnapshot(
+        driver,
+        `Verify Check selfie video screen does not have onfido logo ${lang}`
+      )
       confirm.clickConfirmButton()
       verificationComplete.checkLogoIsHidden()
+      await takePercySnapshot(
+        driver,
+        `Verify Verification complete screen does not have onfido logo ${lang}`
```

Comment:
Wonder if we really need the "Verify ..." bit at the start of each test. It also just happens to sound a bit weird here because the name of the screen itself has "verification" also.
```suggestion
        `Verify Verification complete screen does not have Onfido logo ${lang}`
```